test/integration/CMakeLists.txt

```diff
@@ -9,3 +9,10 @@ add_subdirectory(validation)
 if (USE_BTF)
   add_subdirectory(binary)
 endif()
+
+
+addtest(grpc_retry_test grpc_retry_test.cpp)
+target_link_libraries(grpc_retry_test
+    application
```

Comment:
`application` should not be required here, because only several components are used in the test.